en-US/O.xml

```diff
@@ -244,10 +249,11 @@
 			<term>open source way</term>
 			 <listitem>
 				<para>
-					A phrase that was coined by the Red&nbsp;Hat community and adopted by opensource.com in 2009. It is a reference to an "open source method", as in "Let's develop this project the open source way."
+					A phrase that was coined by the Red&nbsp;Hat community and adopted by opensource.com in 2009.
+          It is a reference to an "open source method", as in "Let's develop this project the open source way."
```

Comment:
Given our guidance to avoid contractions, we should avoid them here in a quoted example.